### PR TITLE
Russian translations

### PR DESCRIPTION
**Pull request by Khudoyshukur**
## 📦 Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Khudoyshukur** [comment]
Russian translations

File: imagepicker/src/main/res/values-ru/strings.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+<resources>
+    <string name="app_name" translatable="false">Image Picker</string>
+    <string name="image_picker_provider_authority_suffix" translatable="false">.imagepicker.provider</string>
+
+    <string name="title_choose_image_provider">Выбирать</string>
+    <string name="action_cancel">Отмена</string>
+    <string name="title_gallery">Галерея</string>
+    <string name="title_camera">Камера</string>
+
+    <string name="permission_camera_denied">![CDATA[Camera требуется разрешение для фотографирования. Разрешите, пожалуйста, Настройки > Разрешения.]]</string>
```
Comment (Dhaval2404) [inline imagepicker/src/main/res/values-ru/strings.xml:11]: Can you please review this once?

File: imagepicker/src/main/res/values-ru/strings.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_name" translatable="false">Image Picker</string>
+    <string name="image_picker_provider_authority_suffix" translatable="false">.imagepicker.provider</string>
+
+    <string name="title_choose_image_provider">Выбирать</string>
```
Comment (palushin86) [inline imagepicker/src/main/res/values-ru/strings.xml:6]: it is worth writing here: 
<string name="title_choose_image_provider">Выбор источника</string>

**Comment by Khudoyshukur** [comment]
Thanks for your comment. I am not a native Russian speaker. I just translated them with the help of online tools. If you are a native I would ask you for checking all the translations...
--
Sent from Mail.ru app for Android Wednesday, 25 August 2021, 07:43PM +05:00 from palushin86  ***@***.*** :

***@***.*** commented on this pull request.

File: imagepicker/src/main/res/values-ru/strings.xml (hunk 1 above)
Comment (palushin86) [inline imagepicker/src/main/res/values-ru/strings.xml:11]: <string name="permission_camera_denied">![CDATA[Для съемки требуется доступ к камере. Пожалуйста, дайте разрешение через Настройки > Разрешения.]]</string>

File: imagepicker/src/main/res/values-ru/strings.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <string name="action_cancel">Отмена</string>
+    <string name="title_gallery">Галерея</string>
+    <string name="title_camera">Камера</string>
+
+    <string name="permission_camera_denied">![CDATA[Camera требуется разрешение для фотографирования. Разрешите, пожалуйста, Настройки > Разрешения.]]</string>
+
+    <string name="error_failed_to_create_camera_image_file">Не удалось создать файл изображения камеры</string>
```
Comment (palushin86) [inline imagepicker/src/main/res/values-ru/strings.xml:13]: <string name="error_failed_to_create_camera_image_file">Не удалось создать снимок</string>

File: imagepicker/src/main/res/values-ru/strings.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    <string name="error_failed_pick_gallery_image">Не удалось выбрать изображение галереи</string>
```
Comment (palushin86) [inline imagepicker/src/main/res/values-ru/strings.xml:14]: <string name="error_failed_pick_gallery_image">Не удалось выбрать изображение из галереи</string>

File: imagepicker/src/main/res/values-ru/strings.xml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+    <string name="error_failed_to_compress_image">Не удалось сжать изображение</string>
+    <string name="error_task_cancelled">Задача отменена</string>
+    <string name="error_camera_app_not_found">Приложение камеры не найдено</string>
+
+
+    <string name="ucrop_label_original">Оригинал</string>
+    <string name="ucrop_label_edit_photo">Редактировать фотографию</string>
+    <string name="ucrop_menu_crop">Обрезать</string>
+    <string name="ucrop_rotate">Повернуть</string>
+    <string name="ucrop_scale">Шкала</string>
```
Comment (palushin86) [inline imagepicker/src/main/res/values-ru/strings.xml:25]: <string name="ucrop_scale">Масштаб</string>

File: imagepicker/src/main/res/values-ru/strings.xml [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+    <string name="error_failed_to_create_camera_image_file">Не удалось создать файл изображения камеры</string>
+    <string name="error_failed_pick_gallery_image">Не удалось выбрать изображение галереи</string>
+    <string name="error_failed_to_crop_image">Не удалось обрезать изображение</string>
+    <string name="error_failed_to_compress_image">Не удалось сжать изображение</string>
+    <string name="error_task_cancelled">Задача отменена</string>
+    <string name="error_camera_app_not_found">Приложение камеры не найдено</string>
+
+
+    <string name="ucrop_label_original">Оригинал</string>
+    <string name="ucrop_label_edit_photo">Редактировать фотографию</string>
```
Comment (palushin86) [inline imagepicker/src/main/res/values-ru/strings.xml:22]: Фоторедактор

**Review by palushin86 — CHANGES_REQUESTED** [review]
I checked all the translations at your request. I suggest doing this

**Comment by Orange1353** [comment]
what do you thing about accept this issue? it will be very helpful, thanks <З

**Comment by Orange1353** [comment]
/

**Pull request closed without merging**